Force lftp to use just ncurses (rather than both Solaris curses and ncurses)
because of the conflicting definition of setupterm:

/usr/include/curses.h
  extern int setupterm(char *, int, int *);

/usr/include/ncurses/term.h
  extern NCURSES_EXPORT(int) setupterm (NCURSES_CONST char *,int,int *);

This appears to be a problem in the Solaris curses header file so not
sending upstream.

--- src/lftp_tinfo.cc.orig	2017-05-09 07:17:26.390916385 +0000
+++ src/lftp_tinfo.cc	2017-05-09 07:30:44.168086861 +0000
@@ -20,14 +20,7 @@
 #include <config.h>
 
 extern "C" {
-#if defined(HAVE_CURSES_H)
-# include <curses.h>
-# if defined(HAVE_TERM_H)
-#  include <term.h>
-# elif defined(HAVE_NCURSES_TERM_H)
-#  include <ncurses/term.h>
-# endif
-#elif defined(HAVE_NCURSES_CURSES_H)
+#if defined(HAVE_NCURSES_CURSES_H)
 # include <ncurses/curses.h>
 # if defined(HAVE_NCURSES_TERM_H)
 #  include <ncurses/term.h>
